components/clubs.jsx

```diff
@@ -0,0 +1,164 @@
+var React = require('react');
```

Comment:
Duh, this is back again?  I feel like I've deleted this file (along with other ones) a million times.  Anyways I'll do it again.